include/psa/crypto_sizes.h

```diff
@@ -971,7 +971,8 @@
  */
 #define PSA_CIPHER_ENCRYPT_OUTPUT_MAX_SIZE(input_length)                        \
     (PSA_ROUND_UP_TO_MULTIPLE(PSA_BLOCK_CIPHER_BLOCK_MAX_SIZE,                  \
-                              (input_length) + PSA_BLOCK_CIPHER_BLOCK_MAX_SIZE))
+                              (input_length) + 1) +                             \
+     PSA_BLOCK_CIPHER_BLOCK_MAX_SIZE)
```

Comment:
Again, I think this is correct in practice, as but as a matter of principle and for readability, we should use `PSA_CIPHER_IV_MAX_SIZE` here.